docs/source/examples/mg-mode-part-ii.ipynb

```diff
@@ -0,0 +1,1661 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7fcd7c0ef3c8>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "root_universe.plot(center=(0., 0., 0.), width=(21.42, 21.42), pixels=(500, 500), color_by='material')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Looks good!\n",
+    "\n",
+    "We now must create a geometry that is assigned a root universe and export it to XML."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Create Geometry and set root Universe\n",
+    "geometry = openmc.Geometry()\n",
+    "geometry.root_universe = root_universe\n",
+    "# Export to \"geometry.xml\"\n",
+    "geometry.export_to_xml()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With the geometry and materials finished, we now just need to define simulation parameters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# OpenMC simulation parameters\n",
+    "batches = 1000\n",
+    "inactive = 100\n",
```

Comment:
Reduce the number of inactive batches to 10 - it doesn't even take this many batches to converge the Shannon entropy for this simple fuel assembly. This will help speed things up for this notebook, even if only slightly.